client-network/src/network_ops/default_network_ops_client.rs

```diff
@@ -221,6 +221,53 @@ where
         Ok(tx_aux)
     }
 
+    fn create_unjail_transaction(
+        &self,
+        name: &str,
+        passphrase: &SecUtf8,
+        from_address: &StakedStateAddress,
+        attributes: StakedStateOpAttributes,
+    ) -> Result<TxAux> {
+        let staked_state = self.get_staked_state(name, passphrase, from_address)?;
+        let nonce = staked_state.nonce;
+
+        let transaction = UnjailTx {
+            nonce,
+            address: *from_address,
+            attributes,
+        };
+
+        let public_key = match from_address {
+            StakedStateAddress::BasicRedeem(ref redeem_address) => self
+                .wallet_client
+                .find_staking_key(name, passphrase, redeem_address)?
+                .chain(|| {
+                    (
+                        ErrorKind::InvalidInput,
+                        "Address not found in current wallet",
+                    )
+                })?,
+        };
+        let private_key = self
+            .wallet_client
+            .private_key(passphrase, &public_key)?
+            .chain(|| {
+                (
+                    ErrorKind::InvalidInput,
+                    "Not able to find private key for given address in current wallet",
+                )
+            })?;
+
+        let signature = private_key
+            .sign(transaction.id())
+            .map(StakedStateOpWitness::new)?;
+
+        let signed_transaction = SignedTransaction::UnjailTransaction(transaction, signature);
+        let tx_aux = self.transaction_cipher.encrypt(signed_transaction)?;
```

Comment:
no need to call this, one can construct the TxAux directly

Reply:
removed